### DISCUSSION TITLE
Delta 3Phase useage

### DISCUSSION THREAD

**Discussion by Ironhydroxide** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#102] [Q&A]
I'm looking to monitor power usage for my Delta 3Phase connection (with high leg).  

In another discussion I read:
> If you are going for accuracy here is what you would need:
> 
> 1 main board - 1 add-on board
> 3 AC transformers to bring down the voltage to 9VAC
> Cut JP12 & JP13 on the back of the main board
> Cut 2 bottom (long) pins on the add-on board (2 that the ESP32 does not plug into)
> Add connectors to the main board and add-on board for VA2-/VA2+ (these can be supplied)
> This will allow you to have CT1-CT3 in phase with phase 1, CT4-CT6 in phase with phase 2, and the whole add-on board in phase with phase 3. This method isn't documented yet, but others have done this with success.

But this doesn't really answer how to wire the transformers.  
Do I wire transformers from each leg to the "orange Leg"? (ie center tapped neutral between A-C)? 
Do I wire transformers from each leg to the following leg? (ie A-B, B-C, C-A)

I can't say I understand how to change all the settings, but reading through the datasheet there's a setting for 3P3W.

> This bit defines the voltage/current phase sequence detection mode:
> 0: 3P4W (default)
> 1: 3P3W (Ua is Uab, Uc is Ucb, Ub is not used)

which seems to point to a star/wye configuration as default, but Delta configuration is possible.   How would one set this setting? (i plan on using ESPHome integration with home assistant)

**Comment by CircuitSetup** [comment]
What you describe hardware wise would be the easiest way to monitor 3 phase, yes. 

For your 3 phase delta system, is the orange leg a high leg? True delta systems wouldn't have a neutral, hence the 3 wire part.

In ESPHome, there is a setting (`current_phases`) that defaults to 3P4W, but this doesn't matter, since ESPHome is not using the energy sum total registers on the metering chips, and 2 chips are used per board. This is really only relevant when using 1 chip, and using the chip's energy sum total registers.

**Reply by Ironhydroxide** [reply]
The "orange Leg" is the neutral between A and C legs (A-Orange =120vac, C-Orange = 120vac, A-C = 240vac, B-Orange = 208vac)

So, it's  a High-Leg Delta. (Delta, with some 120vac available) 

Seeing as it's 2 chips/board, then it doesn't really matter, as they'll calculate per phase, instead of combined?
